website/docs/guides/cts/ttl-index.json

```
@@ -0,0 +1,59 @@
+{
+  "create-ttl-index": {
```

```suggestion
  "1-create-ttl-index": {
```

In that case, it was called correctly, we should keep that name